src/lib/operators/abstract_aggregate_operator.cpp

```diff
@@ -29,24 +29,24 @@ const std::vector<std::shared_ptr<AggregateExpression>>& AbstractAggregateOperat
 const std::vector<ColumnID>& AbstractAggregateOperator::groupby_column_ids() const { return _groupby_column_ids; }
 
 std::string AbstractAggregateOperator::description(DescriptionMode description_mode) const {
-  const auto* const separator = description_mode == DescriptionMode::SingleLine ? " " : "\n";
+  const char separator = (description_mode == DescriptionMode::SingleLine ? ' ' : '\n');
 
   std::stringstream desc;
-  desc << AbstractOperator::description(description_mode) << separator << "GroupBy ColumnIDs: ";
+  desc << AbstractOperator::description(description_mode) << separator;
+  desc << "GroupBy ColumnIDs: {";
   for (size_t groupby_column_idx = 0; groupby_column_idx < _groupby_column_ids.size(); ++groupby_column_idx) {
     desc << _groupby_column_ids[groupby_column_idx];
 
     if (groupby_column_idx + 1 < _groupby_column_ids.size()) {
```

Comment:
What about moving the `if` to the front with `if (groupby_column_idx > 0)` and prepend the comma?